src/server/routes/plywood/plywood.ts

```diff
@@ -98,9 +97,9 @@ export function plywoodRouter(settingsManager: Pick<SettingsManager, "anchorPath
       };
       res.json(reply);
     } catch (error) {
-      console.log("error:", error.message);
+      logger.log("error:", error.message);
       if (error.hasOwnProperty("stack")) {
-        console.log((error as any).stack);
+        logger.log((error as any).stack);
```

Comment:
I would build single log entry from error.message and error.stack (if exists). With errors from many concurrent users it could be hard to correlate errors from single client.